### PR TITLE
Fix/carla carma agent remove autoware

### PR DESCRIPTION
**Pull request by chengyuan0124**
# PR Details
## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: carla-carma-integration/carla-carma-agent/agent/config/sensors.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
+{
+    "sensors": [
```
Comment (kjrush) [inline carla-carma-integration/carla-carma-agent/agent/config/sensors.json:2]: Is this configuring the sensor simulation in CARLA? If so, do these sensor positions need to be correlated with the positions in the CARMA simulation UDRF file?

File: carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch [hunk 2]
```diff
@@ -20,28 +20,37 @@
 -->
 
 <launch>
+  <param name="use_sim_time" value="True"/>
```
Comment (kjrush) [inline carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch:23]: Had to double check that boolean params aren't case sensitive but that appears to be the case, so this should be fine. Normally I'm used to seeing all lower case for bools.

File: carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch [hunk 3]
```diff
@@ -20,28 +20,37 @@
...
   <arg name='host' default='localhost'/>
   <arg name='port' default='2000'/>
   <arg name='town' default='Town02'/>
   <arg name='vehicle_filter' default='vehicle.toyota.prius'/>
   <arg name='role_name' default='ego_vehicle'/>
   <arg name='spawn_point' default=''/>
 
-  <!-- carla autoware bridge -->
+  <arg name='speed_Kp' default='0.05'/>
```
Comment (kjrush) [inline carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch:31]: Are these PID params separate from the CARMA controller ones? Should they be the same?

File: carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch (hunk 2 above)
Comment (chengyuan0124) [reply carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch]: The boolean value has been corrected to lowercase

File: carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch (hunk 3 above)
Comment (chengyuan0124) [reply carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch:31]: The PID default value has been set as the same value from CARMA [platooning_control config](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/platooning_control/config/parameters.yaml)

File: carla-carma-integration/carla-carma-agent/agent/config/sensors.json (hunk 1 above)
Comment (chengyuan0124) [reply carla-carma-integration/carla-carma-agent/agent/config/sensors.json:2]: The sensor file has been modified according to the carma.urdf file

**Review by kjrush — APPROVED** [review]

**Pull request merged**